tests/test_task_details.py

```diff
@@ -0,0 +1,28 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from pages.home import HomePage
+
+
+class TestTaskDetails():
+
+    def test_that_person_can_view_task_details(self, base_url, selenium):
+        home_page = HomePage(base_url, selenium).open()
+        assert not home_page.is_user_logged_in
+
+        task = home_page.suggested_first_tasks[0]
+        task_name = task.name
+        task_details = task.click()
+        assert not task_details.is_user_logged_in
```

Comment:
I'm not sure why I didn't comment on this before, but this assert is unnecessary.